PayItForward.ConsoleClient.UnitTests/Web/Controllers/HomeController/Index_Should.cs

```diff
@@ -89,12 +92,218 @@ public void ReturnIndexViewModel_WithListCategoriesViewModelAsTypeOfCategoriesPr
                 .Returns(HomeController_Stubs.GetTestCategoriesListWithCategoryDTOs());
 
             // Act
-            var result = this.homeController.Index(1, categoryname, "/Home/Index/");
+            var result = this.homeController.Index(1, categoryname, subTitle);
 
             // Assert
             var viewResult = Assert.IsType<ViewResult>(result);
             var indexViewModel = (IndexViewModel)viewResult.ViewData.Model;
             Assert.IsAssignableFrom<PayItForward.Web.Models.CategoryViewModels.ListCategoriesViewModel[]>(indexViewModel.Categories);
         }
+
+        [Fact]
+        public void ReturnIndexViewModel_WithCorrectCategoryNames()
+        {
+            // Arrange
+            var categoryname = string.Empty;
+            var subTitle = string.Empty;
+            var dtoStories = HomeController_Stubs.GetTestListOfStoryDtos();
+            var dtoCategories = HomeController_Stubs.GetTestCategoriesListWithCategoryDTOs();
+            var expectedCategoriesViewModels = dtoCategories.Select(x => new ListCategoriesViewModel()
+            {
+                Name = x.Name
+            }).ToList();
+
+            this.storiesService.Setup(
```

Comment:
This could be moved to separate private method, because this code is duplicated.